guide/onboarding/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
+---
+
+<!--
+
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
```

Comment:
```suggestion
Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard, as they might assume that the patterns and habits from prior experience will apply. Convincing them to use a different financial system might be better in the product. 
```

Not totally clear what you mean by `might be better in the product.`. As opposed to what?

Reply:
Trying to get across the idea that the experience of using a bitcoin product should be self-evident that it is better than the traditional financial system.